test/elixir_analyzer/exercise_test/feature_test.exs

```diff
@@ -382,4 +382,227 @@ defmodule ElixirAnalyzer.ExerciseTest.FeatureTest do
       ]
     end
   end
+
+  describe "types of blocks" do
+    defmodule Blocks do
+      use ElixirAnalyzer.ExerciseTest
+
+      feature "single line" do
+        comment "single line"
+
+        form do
+          :a
+        end
+      end
+
+      feature "two lines" do
+        comment "two lines"
+
+        form do
+          :a
+          :b
+        end
+      end
+
+      feature "single line block" do
+        comment "single line block"
+
+        form do
+          !:a
+        end
+      end
+    end
+
+    defmodule BlocksTest do
+      use ElixirAnalyzer.ExerciseTestCase,
+        exercise_test_module: Blocks
+
+      test_exercise_analysis "empty",
+        comments: ["single line block", "two lines", "single line"] do
+        defmodule Module do
+        end
+      end
+
+      test_exercise_analysis "two lines",
+        comments: ["single line block"],
+        comments_exclude: ["single line", "two lines"] do
+        defmodule Module do
+          def foo do
+            :a
+            :b
+          end
+        end
+      end
+
+      test_exercise_analysis "single line block",
+        comments: ["two lines"],
+        comments_exclude: ["single line", "single line block"] do
+        defmodule Module do
+          def foo do
+            !:a
+          end
+        end
+      end
+    end
+  end
+
+  describe "other features" do
+    defmodule Coverage do
+      use ElixirAnalyzer.ExerciseTest
+
+      feature "skip" do
+        comment "skip"
+        status :skip
```

Comment:
Do we really need to keep that `status` field? It might have been useful for debugging, but we don't actually use it anywhere in the code. I think it's commented out in `two-fer`.

Reply:
My guess is that it would be safe to remove `status`. @neenjaw do you have any thoughts about this?